### PR TITLE
bleeding blockquote over to post meta bug fix

### PR DESCRIPTION
**Pull request by maharzan**
Trying to fix blockquote bleeding over to post meta bug ref: https://github.com/WordPress/twentysixteen/issues/32

I am no JS wiz and I am pretty sure there are better ways to organize and do things. I just copied over the big image code and tried to adjust.

Here are the screenshots:

blockquote-fix2

blockquote-fix1

### CODE REVIEW

**Comment by tywayne** [comment]
Tested this and can confirm that the new function works as expected to adjust a blockquote in the same way we do for images. I did have a couple thoughts to throw out for discussion. 
1. Currently `bigImageClass()` is triggered in `$( document ).ready()` as well as on a resize event. Should the same be done for the new `blockquoteFix()` ?
2. Alternatively, considering these functions are very similar and slightly repetitive, would it be prudent to combine into one function that handles both elements that are offset left?

**Comment by maharzan** [comment]
I have tried to merge the 2 functions into one. I am sure there can be more improvements. :)

**Comment by maharzan** [comment]
Fixes #32

File: js/functions.js [hunk 1]
```diff
@@ -109,44 +109,58 @@
...
 
-	// Add a class to big image and caption larger than or equal to 840px.
-	function bigImageClass() {
+	// Preventing left offseted elements from bleeding into the post meta
+	function offsetLeft( param ) {
 		if ( $body.hasClass( 'page' ) || $body.hasClass( 'search' ) || $body.hasClass( 'single-attachment' ) || $body.hasClass( 'error404' ) ) {
 			return;
 		}
 
-		// jscs:disable
```
Comment (iamtakashi) [inline js/functions.js:138]: We want to keep this comment as per #160.

File: js/functions.js [hunk 2]
```diff
@@ -109,44 +109,58 @@
...
 		var entryContent = $( '.entry-content' );
-		// jscs:enable
```
Comment (iamtakashi) [inline js/functions.js:140]: The same as above.

File: js/functions.js [hunk 3]
```diff
@@ -109,44 +109,58 @@
...
+				var caption              = element.closest( 'figure' ),
+					newImg               = new Image();
+
+				newImg.src = element.attr( 'src' );
+
+				$( newImg ).load( function() {
+					var imgWidth = newImg.width;
+
+					if ( elementPosTop > entryFooterPosBottom ) {
+						if ( 840 <= imgWidth  ) {
```
Comment (iamtakashi) [inline js/functions.js]: Let's flip the yoda conditionals as per #196.

File: js/functions.js [hunk 4]
```diff
@@ -109,44 +109,58 @@
...
+							element.addClass( 'size-big' );
+						}
+
+						if ( caption.hasClass( 'wp-caption' ) && 840 <= imgWidth  ) {
```
Comment (iamtakashi) [inline js/functions.js]: The same as above.

File: style.css [hunk 5]
```diff
@@ -3376,8 +3376,12 @@ p > video {
 		width: 71.42857144%;
 	}
 
-	body:not(.search-results) .type-post .entry-content blockquote.alignleft {
+	body:not(.search-results) .type-post .entry-content blockquote.alignleft.farleft {
 		margin-left: -40%;
+	}
+
+	body:not(.search-results) .type-post .entry-content blockquote.alignleft {
```
Comment (iamtakashi) [inline style.css]: I think we should make only `.farleft` to be wider: `(60% - 1.4736842105em)`, and then we don't need to reset the left margin. We can also remove the edit above.

**Comment by maharzan** [comment]
Ok.. hopefully, its all good with the checks as well. :)

**Pull request merged**